azext_iot/operations/digitaltwin.py

```diff
@@ -199,12 +199,12 @@ def iot_digitaltwin_monitor_events(cmd, device_id=None, device_query=None, inter
 
         if interface:
             target_interfaces.append(target_interface)
-            interface_id = target_interface['urn_id']
+            interface_id = interface
```

Comment:
Can we rename this to interface_name as well?

Reply:
yea, change the variable plumbing so its not ambiguous

Reply:
Changed and updated in related files too.